test/e2e/volumes.go

```diff
@@ -169,6 +169,9 @@ func volumeTestCleanup(client *client.Client, config VolumeTestConfig) {
 		expectNoError(err, "Failed to delete client pod: %v", err)
 	}
 
+	By("sleeping a bit so client can stop and unmount")
+	time.Sleep(20 * time.Second)
+
```

Comment:
@rootfs can you (1) reference the PR that will make this unnecessary (2) add a comment on why we cannot delete the podClient until after this sleep occurs (like you explained to me on the phone :))